docs/multicluster/upgrade.md

```diff
@@ -37,6 +37,45 @@ for the feature in new version.
 It should have no impact during upgrade to those imported resources like Service, Endpoints
 or AntreaClusterNetworkPolicy.
 
+## Upgrade from a version prior to v1.13
+
+Prior to Antrea v1.13, the `ClusterClaim` CRD is used to define both the Cluster ID and
```

Comment:
Maybe we should say "both the local cluster ID and the ClusterSet ID".

Reply:
Updated.